app/views/survey/new.erb

```diff
@@ -1 +1 @@
-<h1>new</h1>
+
```

Comment:
you might be having merge conflicts here because I made the survey new erb file and now github doesnt know how to merge this into that new file
:diamond_shape_with_a_dot_inside: :icecream: